website_animate/static/src/js/o_animate.editor.js

```diff
@@ -34,7 +34,7 @@ sOptions.registry.o_animate = sOptions.Class.extend({
                         'visibility': '',
                     });
         if (this.$target.hasClass('o_animate')) {
-            this.$target.css('animation-play-state', 'paused');
+            this.$target.css('visibility', 'visible');
```

Comment:
I doubt this is the right fix. The o_animate class sets `visibility: hidden` but once the snippet is saved we cancel that via inline style? :thinking: :thinking: